app/test_app/functional/api/test_transfer_account_api.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from server.utils.auth import get_complete_auth_token
+
+
+@pytest.mark.parametrize("transfer_account_id_accessor, tier, status_code", [
+    (lambda u: u.transfer_account.id, None, 401),
+    (lambda u: u.transfer_account.id, 'superadmin', 201),  # todo(#320): this should be a 200 Status code
+    (lambda u: u.transfer_account.id, 'view', 201),  # todo(#320): this should be a 200 Status code
+    (lambda o: 1222103, 'superadmin', 400),  # todo(#320): this should be a 404 Status code
```

Comment:
adding todo's for #320

Reply:
any reason not to just change them in this pr?

Reply:
Trying to keep each PR [atomic](https://en.wikipedia.org/wiki/Atomic_commit)